builder/build_server.py

```diff
@@ -100,18 +113,12 @@
                         eval_queue.send_message(MessageBody=json.dumps(eval_message))
 
                     # send email
-                    user = m.getModelUserByMid(model_id)[1]
-                    if "smtp_user" in config and config["smtp_user"] != "":
-                        mail_session = mail.get_mail_session(
-                            host=config["smtp_host"],
-                            port=config["smtp_port"],
-                            smtp_user=config["smtp_user"],
-                            smtp_secret=config["smtp_secret"],
-                        )
+                    if mail_session:
+                        user = m.getModelUserByMid(model_id)[1]
```

Comment:
I don't understand why we're picking the [1] here? Is that supposed to be a 0 actually? Can we clean up this code a bit more, i.e. if this method returns multiple things then you could do `_, user = m.getModelUserByMid()`?